contracts/javascore/lib/src/main/java/ibc/icon/interfaces/IMock.java

```diff
@@ -0,0 +1,8 @@
+package ibc.icon.interfaces;
+
+import foundation.icon.score.client.ScoreInterface;
+
+@ScoreInterface
+public interface IMock {
```

Comment:
Is this used anywhere?

Reply:
Yes it is used create empty MockContract for contract address.

Reply:
You can use sm.createScoreAccount for this,  MockContracts are only useful to mock and verify method calls.